### DISCUSSION TITLE
HA Network Storage of a HA running in a docker container

### DISCUSSION THREAD

**Discussion by janusn** [blakeblackshear/frigate#8466] [Ask A Question] [answered]
Regarding the document on [HA Network Storage](https://docs.frigate.video/guides/ha_network_storage), a network storage from frigate can be mounted on HA OS.

I have a HA installed in a docker container. Frigate is integrated to HA via [Frigate integration](https://github.com/blakeblackshear/frigate-hass-integration). In HA, Frigate is shown in the Media tab. There are 3 folders inside, Clips…, Recordings… and Snapshots… with respective files and folders under them.

My questions are,
1. what is HA Network Storage used for?
2. How does it benefit Frigate?
3. How to configure HA in a container with Network Storage?

Thanks in advance.

**Comment by NickM-27** [comment, marked as answer]
> what is HA Network Storage used for?

It is only relevant for HA OS because that is the OS running the frigate container itself. This allows a user to store frigate recordings on some other devices like a NAS with more storage space. 

> How does it benefit Frigate?

More storage space

> How to configure HA in a container with Network Storage?

You can't and it's irrelevant because HA container is not running frigate or storing any Frigate data.

**Reply by janusn** [reply]
Thank you. 😊 

Maybe the document page could make it more clearly for novices like me. 😳
